types/publish.mjs

```diff
@@ -98,20 +98,6 @@ ${MODULES_PLACEHOLDER}
 
 const TYPES_DIR = path.join('types', 'stable');
 
-// These modules need to be copied over *and* post-processed: they need to be
-// locally-importable as regular modules, but `tsc` will not move them over
-// itself, but unlike some modules which need to be left unchanged (e.g. the
-// `*-ext.d.ts` modules) these still need the module wrapper added.
-const HAND_COPIED_BUT_NEEDS_POSTPROCESSING = [
-  'ember/version.d.ts',
-  '@ember/-internals/glimmer/lib/templates/empty.d.ts',
-  '@ember/-internals/glimmer/lib/templates/input.d.ts',
-  '@ember/-internals/glimmer/lib/templates/link-to.d.ts',
-  '@ember/-internals/glimmer/lib/templates/outlet.d.ts',
-  '@ember/-internals/glimmer/lib/templates/root.d.ts',
-  '@ember/-internals/glimmer/lib/templates/textarea.d.ts',
-];
```

Comment:
All of this was helping typescript deal with fake files that don't exist. But now all of them are real files.